modules/apps/push-notifications/push-notifications-sender/push-notifications-sender-apple/src/main/java/com/liferay/push/notifications/sender/apple/internal/messaging/ApplePushNotificationsResponseMessageListener.java

```diff
@@ -46,33 +34,7 @@ protected void doReceive(Message message) throws Exception {
 			return;
 		}
 
-		AppleResponse appleResponse = (AppleResponse)response;
```

Comment:
Why did you remove this?

Reply:
`DeliveryError.INVALID_TOKEN` was of the java-apns library. I looked for a way to handle Exceptions with Pushy but AFAIK it only throws exceptions, it doesn't handle it.
So, I decided to throw a `PushNotificationException` with the status of the response.

Reply:
👍